frontend/js/utils.js

```diff
@@ -106,3 +106,19 @@ export function isControlField(field) {
 export function isCoreError(error) {
   return ([EvalError, RangeError, URIError, TypeError, SyntaxError, ReferenceError].some(errorType => error instanceof errorType));
 }
+
+export function sortSubfields(a, b) {
```

Comment:
Subfields shouldn't be sorted in existing records. I think we already have this function implemented elsewhere and used when creating new records in some application.